src/components/10-atoms/toggle-switch/index.js

```diff
@@ -56,7 +58,12 @@ class AXAToggleSwitch extends LitElement {
   }
 
   render() {
-    const { active, disabled, handleChange } = this;
+    const { active, disabled, error, handleChange } = this;
+
+    const classes = {
+      'a-toggle-switch__error-message': true,
```

Comment:
If this is always true, there is no need to define it here. Just put it into the template directly.